documents/openapi/openapi.yaml

```diff
@@ -1149,7 +1132,61 @@ paths:
           $ref: '#/components/responses/NotFound'
         '500':
           $ref: '#/components/responses/InternalServerError'
-
+  /inbound-number:
+    get:
+      summary: Retrieve an inbound number
+      parameters: {} # TODO
```

Comment:
Let's please stop adding TODOs to the codebase. See my comment above, we might be better off handling all the swagger changes on a new ticket to avoid any TODOs.

Reply:
The parameter line is not necessary.  I deleted it.